snuba/utils/streams/topics.py

```diff
@@ -30,6 +30,7 @@ class Topic(Enum):
     DEAD_LETTER_QUEUE_INSERTS = "snuba-dead-letter-inserts"
     DEAD_LETTER_METRICS = "snuba-dead-letter-metrics"
     DEAD_LETTER_SESSIONS = "snuba-dead-letter-sessions"
+    EXPERIMENTAL = "does-not-exist"
```

Comment:
This will be automatically created in self hosted and single tenant. Can we find a way to avoid that?

Reply:
as in an experimental kafka topic will automatically be created? Where is that done?

Reply:
No, everything on this list does. It doesn't know because experimental is not marked on this topic, that's only a property of the dataset. It's done by the bootstrap script.

Reply:
One option for a quick test would be to just point your storage at some other real topic already present here. It should be harmless anyway since it doesn't produce or consume anything from there.